### DISCUSSION TITLE
Firestick/android client improvements lacking

### DISCUSSION THREAD

**Discussion by chuckkay** [jellyfin/jellyfin-androidtv#2926] [General]
in direct contrast to the backend server, and how smooth and feature Rich, it is, the android client player is seriously lacking in features, it seems the developers are more focused on settings to control backend functionality, as opposed to improving the client environment. Specifically the play back options are basic at best, load times and artwork reliability are extremely buggy. Features you would expect on a smart TV client are missing, but available on Web and PC clients, so many features expected for a quality home theater experience are missing or flawed and instead we are given a total mess tiles and useless features most of which are so unrefined that they are anoying. I’m very surprised to find nobody has forked this client and improved on it. 
 To summarize my opinion  i think jellyfin server is great, but the android client is embarrassing

**Comment by nielsvanvelzen** [comment]
If you're not satisfied with my work (that I do for free in my spare time btw) you can use different software, or feel free to work on those issues yourself. I have no obligation nor motivation to help you with this attitude.

**Reply by chuckkay** [reply]
previous work appreciated, but swathing from plex to jellyfin and being pleased with server, only to find the player maintained separately, and by one person is disappointing, typical response from someone overwhelmed and not focused on the concerns of the community, like i said surprised no fork, as your initial work is admirable, but now  practically stalled as not focused on implementing missing enhancements. requested for so long.. a shame i have no alternatives after already having migrated to jellyfin .

**Reply by nielsvanvelzen** [reply]
You must be fun at parties.

**Comment by thornbill** [comment]
Frankly what I find embarrassing is the frequent attacks and hostility directed towards this client and the **one volunteer** who primarily maintains it. Of course constructive feedback is welcome, but often, especially lately, it's just been excessive negativity and hatefulness. 😖

**Comment by chuckkay** [comment]
> If you're not satisfied with my work (that I do for free in my spare time btw) you can use different software, or feel free to work on those issues yourself. I have no obligation nor motivation to help you with this attitude.



> Frankly what I find embarrassing is the frequent attacks and hostility directed towards this client and the **one volunteer** who primarily maintains it. Of course constructive feedback is welcome, but often, especially lately, it's just been excessive negativity and hatefulness. 😖

not hostile just disappointed with the stagnation
